### PR TITLE
549 send metadata from sample data to eq for routing piping and linking

### PR DESCRIPTION
**Pull request by LukeLoze1**
# Motivation and Context
To use collectionInstrument level launch settings

# What has changed
Added a test for this

# How to test?
BRANCH_NAME=549-send-metadata-from-sample-data-to-eq-for-routing-piping-and-linking ./checkout_and_build_pr.sh

# Links
https://trello.com/b/VS0ZyRmj/srm

### CODE REVIEW

File: acceptance_tests/features/deactivate_uac.feature [hunk 1]
```diff
@@ -1,15 +1,15 @@
-
 Feature: deactivate UACs for a case
 
   @regression
   Scenario: A case is loaded, action rule is triggered and UAC is deactivated for that case
     Given sample file "sample_1_limited_address_fields.csv" is loaded successfully
     And an export file template has been created with template ["__uac__"]
-    And an export file action rule has been created
+    When an export file action rule has been created
```
Comment (ryangrundy7) [inline acceptance_tests/features/deactivate_uac.feature:7]: Is this meant to be `when`

**Review by ryangrundy7 — APPROVED** [review]
Looks good to me 👍

File: resources/validation_rules/PHM_made_up_settings_2.json [hunk 2]
```diff
@@ -0,0 +1,188 @@
+[
```
Comment (liamtoozer) [inline resources/validation_rules/PHM_made_up_settings_2.json:1]: 🎨 Worth calling this file something like `PHM_validation_rules` or something? The other PHM file is a prototype, so should be fine?

**Review by liamtoozer — APPROVED** [review]
LGTM 👍

File: resources/validation_rules/PHM_made_up_settings_2.json (hunk 2 above)
Comment (LukeLoze1) [reply resources/validation_rules/PHM_made_up_settings_2.json:1]: I might change this on the next ticket I'm on

**Pull request merged**